diff --git a/lib/theme/types/MantineColor.d.ts b/lib/theme/types/MantineColor.d.ts
index 49cc8560e6fbbd25f3a260c3da15f54c705a5dec..209bcb810bb727fc922b9033c39301032449a514 100644
--- a/lib/theme/types/MantineColor.d.ts
+++ b/lib/theme/types/MantineColor.d.ts
@@ -1,6 +1,6 @@
 import { Tuple } from './Tuple';
 export type DefaultMantineColor = 'dark' | 'gray' | 'red' | 'pink' | 'grape' | 'violet' | 'indigo' | 'blue' | 'cyan' | 'green' | 'lime' | 'yellow' | 'orange' | 'teal' | (string & {});
-export type MantineThemeColorsOverride = {};
+export interface MantineThemeColorsOverride {};
 export type MantineThemeColors = MantineThemeColorsOverride extends {
     colors: Record<infer CustomColors, Tuple<string, 10>>;
 } ? Record<CustomColors, Tuple<string, 10>> : Record<DefaultMantineColor, Tuple<string, 10>>;
