diff --git a/cjs/Loader/loaders/Oval.js b/cjs/Loader/loaders/Oval.js
index 4699137ec171fd80020a519c348b99f6407b5e1e..8858da0dbb15a672c2e06fd3078a7920128acec8 100644
--- a/cjs/Loader/loaders/Oval.js
+++ b/cjs/Loader/loaders/Oval.js
@@ -41,30 +41,23 @@ function Oval(_a) {
   return /* @__PURE__ */ React__default.createElement("svg", __spreadValues({
     width: `${size}px`,
     height: `${size}px`,
-    viewBox: "0 0 38 38",
+    viewBox: "0 0 40 40",
     xmlns: "http://www.w3.org/2000/svg",
-    stroke: color
+    stroke: color,
+    className: _b.className + ' mantine-loader-root'
   }, others), /* @__PURE__ */ React__default.createElement("g", {
     fill: "none",
     fillRule: "evenodd"
   }, /* @__PURE__ */ React__default.createElement("g", {
-    transform: "translate(2.5 2.5)",
     strokeWidth: "5"
   }, /* @__PURE__ */ React__default.createElement("circle", {
     strokeOpacity: ".5",
-    cx: "16",
-    cy: "16",
+    cx: "50%",
+    cy: "50%",
     r: "16"
   }), /* @__PURE__ */ React__default.createElement("path", {
-    d: "M32 16c0-9.94-8.06-16-16-16"
-  }, /* @__PURE__ */ React__default.createElement("animateTransform", {
-    attributeName: "transform",
-    type: "rotate",
-    from: "0 16 16",
-    to: "360 16 16",
-    dur: "1s",
-    repeatCount: "indefinite"
-  })))));
+    d: "M36 20c0-9.94-8.06-16-16-16"
+  }))));
 }
 
 exports.Oval = Oval;
diff --git a/esm/Loader/loaders/Oval.js b/esm/Loader/loaders/Oval.js
index 7b6db7f8de538ae3cc8ba2632adbabbd433c8d43..b813700bb1fe78040b7fd38caf554ce903d4352e 100644
--- a/esm/Loader/loaders/Oval.js
+++ b/esm/Loader/loaders/Oval.js
@@ -29,34 +29,27 @@ var __objRest = (source, exclude) => {
   return target;
 };
 function Oval(_a) {
-  var _b = _a, { size, color } = _b, others = __objRest(_b, ["size", "color"]);
+  var _b = _a, { size, color } = _b, others = __objRest(_b, ["size", "color", "className"]);
   return /* @__PURE__ */ React.createElement("svg", __spreadValues({
     width: `${size}px`,
     height: `${size}px`,
-    viewBox: "0 0 38 38",
+    viewBox: "0 0 40 40",
     xmlns: "http://www.w3.org/2000/svg",
-    stroke: color
+    stroke: color,
+    className: _b.className + ' mantine-loader-root'
   }, others), /* @__PURE__ */ React.createElement("g", {
     fill: "none",
     fillRule: "evenodd"
   }, /* @__PURE__ */ React.createElement("g", {
-    transform: "translate(2.5 2.5)",
     strokeWidth: "5"
   }, /* @__PURE__ */ React.createElement("circle", {
     strokeOpacity: ".5",
-    cx: "16",
-    cy: "16",
+    cx: "50%",
+    cy: "50%",
     r: "16"
   }), /* @__PURE__ */ React.createElement("path", {
-    d: "M32 16c0-9.94-8.06-16-16-16"
-  }, /* @__PURE__ */ React.createElement("animateTransform", {
-    attributeName: "transform",
-    type: "rotate",
-    from: "0 16 16",
-    to: "360 16 16",
-    dur: "1s",
-    repeatCount: "indefinite"
-  })))));
+    d: "M36 20c0-9.94-8.06-16-16-16"
+  }))));
 }
 
 export { Oval };
